manifests/n/NextDNS/NextDNS/3.0.0/NextDNS.NextDNS.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: NextDNS.NextDNS
+PackageVersion: 3.0.0
+PackageLocale: en-US
+Publisher: NextDNS
+PackageName: NextDNS Client

```

```suggestion
PackageName: NextDNS
```
This must match Control Panel.